quota/mysqlqm/mysql_quota_test.go

```diff
@@ -304,7 +304,9 @@ func createTree(ctx context.Context, db *sql.DB) (*trillian.Tree, error) {
 	{
 		ls := mysql.NewLogStorage(db, nil)
 		err := ls.ReadWriteTransaction(ctx, tree, func(ctx context.Context, tx storage.LogTreeTX) error {
-			return tx.StoreSignedLogRoot(ctx, trillian.SignedLogRoot{LogId: tree.TreeId, RootHash: []byte{0}, Signature: &sigpb.DigitallySigned{}})
+			return tx.StoreSignedLogRoot(ctx, trillian.SignedLogRoot{
```

Comment:
(nit) Although it's just a test, why not use `types.SerializeKeyHint` here too?

Reply:
I'll use a proper signer.